src/lib/radio/radio.ts

```diff
@@ -427,6 +466,12 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
     this._focusOriginMonitor.focusVia(this._inputElement.nativeElement, this._renderer, 'keyboard');
   }
 
+  _groupValueChanged() {
```

Comment:
I would just call this function `_markForCheck` with a comment like 
```ts
/**
 * Marks the radio button as needing checking for change detection. 
 * This method is exposed because the parent radio group will directly 
 * update bound properties of the radio button.
 */
```